pallets/asset/src/lib.rs

```diff
@@ -256,6 +256,22 @@ pub struct ClassicTickerRegistration {
     pub is_created: bool,
 }
 
+// A value placed in storage that represents the current version of the this storage. This value
+// is used by the `on_runtime_upgrade` logic to determine whether we run storage migration logic.
+// NB If you add a new value, don't forget to update the implementation of `Default`.
```

Comment:
The default should remain `Version::V0`. We should instead change the `build(|_| Version::V1)`.

Reply:
Yep, that comment is not aligned with the impl.